src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                                owl:allValuesFrom soreag:Moho
+                              ] ;
+              rdfs:label "mantle"@en .
+
+
+###  http://sweetontology.net/realmGeol/Member
+soreag:Member rdf:type owl:Class ;
+              rdfs:subClassOf soreag:LithostratigraphicUnit ;
+              dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+              dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ;
+              skos:definition "The formal lithostratigraphic unit next in rank below a formation. (ICS, Online, December 2019)"@en ;
+              skos:prefLabel "member"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Mesosphere
 soreag:Mesosphere rdf:type owl:Class ;
-                owl:equivalentClass soreag:UpperMantleRigid ;
-                rdfs:label "mesosphere"@en .
+                  owl:equivalentClass soreag:UpperMantleRigid ;
+                  rdfs:subClassOf soreag:UpperMantle ;
+                  rdfs:label "mesosphere"@en .
+
+
+###  http://sweetontology.net/realmGeol/MineralProvince
+soreag:MineralProvince rdf:type owl:Class ;
+                       rdfs:subClassOf soreag:GeologicProvince ;
+                       dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                       dcterms:source <http://www.glossaryofgeology.org> ;
+                       skos:altLabel "Metallogenic province"@en ;
+                       skos:definition "An area characterized by a particular assemblage of mineral deposits, or by one or more characteristic types of mineralization. A metallogenic province may have had more than one episode of mineralization.  (Neuendorf et al., 2011)"@en ;
+                       skos:prefLabel "mineral province"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Moho
 soreag:Moho rdf:type owl:Class ;
-          owl:equivalentClass soreag:Mohorovi_i_Discontinuity ;
-          rdfs:subClassOf soreag:GeologicBoundary ;
-          rdfs:label "moho"@en .
+            owl:equivalentClass soreag:Mohorovi_i_Discontinuity ;
+            rdfs:subClassOf soreag:GeologicBoundary ;
+            rdfs:label "moho"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Mohorovi_i_Discontinuity
 soreag:Mohorovi_i_Discontinuity rdf:type owl:Class ;
-                              rdfs:label "mohorovicic discontinuity"@en .
+                                rdfs:subClassOf soreag:GeologicBoundary ;
+                                rdfs:label "mohorovicic discontinuity"@en .
+
+
+###  http://sweetontology.net/realmGeol/Monocline
+soreag:Monocline rdf:type owl:Class ;
+                 rdfs:subClassOf soreag:StructuralFeature ;
+                 dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                 dcterms:source <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+                 skos:definition "A one-limbed flexure on either side of which the strata are horizontal or dip uniformly at low angles. (Allaby, 2013)"@en ;
```

Comment:
sopropm:Flexure is a SolidProp0erty/StrengthProperty.  Obviously not applicable here.  
What is a flexure???  
Isn't a monocline a kind of fold???

Suggest def:
A structural feature consisting of a pair of folds that bound a relatively short, more steeply dipping fold limb (link to def) between two more extensive generally gently dipping longer limbs.

Scope note: e.g. Kaibab monocline (link)(

Reply:
I agree flexure is problematic because it is used to describe folding mechanisms which are independent of the geometry of a fold (monocline or otherwise). However, I think of a monocline as a single structural feature rather than a closely spaced antiform-synform pair in sub-horizontal rocks as this is the way it is represented on most maps.  

Revised suggest def:
A step-like fold consisting of a relatively short steeply dipping limb (link to def) bounded by the axial surfaces of a pair of folds between two more extensive and gently dipping longer limbs. 
Scope note: e.g. Kaibab monocline (link)

Reply:
Better!  Maybe could be simplified further...
A step-like fold consisting of a relatively short steeply dipping limb (link to def)  between two more extensive and gently dipping longer limbs.

Reply:
Are you happy with this @geoderekh

Reply:
yup!